ui/src/admin/components/chronograf/OrganizationsTableRow.js

```diff
@@ -119,8 +131,17 @@ class OrganizationsTableRow extends Component {
 
     return (
       <div className="orgs-table--org">
-        <div className="orgs-table--id">
-          {organization.id}
+        <div className="orgs-table--active">
+          {organization.id === currentOrganization.id
+            ? <button className="btn btn-sm btn-success">
+                <span className="icon checkmark" /> Current
+              </button>
+            : <button
+                className="btn btn-sm btn-default"
+                onClick={this.handleChangeCurrentOrganization()}
```

Comment:
If you don't evaluate this function (remove the `()`) you can remove the first anonymous function in `handleChangeCurrentOrganization`.  So, line 41 would look like `handleChangeCurrentOrganization = async () => {`

Reply:
Ahh good catch, changed